configure.ml

```diff
@@ -1234,23 +1235,11 @@ let write_configml f =
 
   close_out o;
   Unix.chmod f 0o444
-
 let _ = write_configml "config/coq_config.ml"
-
-(** * Symlinks or copies for the checker *)
-
-let _ =
-  let prog, args, prf =
-    if arch = "win32" then "cp", [], ""
-    else "ln", ["-s"], "../" in
-  List.iter (fun file ->
-    ignore(run "rm" ["-f"; "checker/"^file]);
-    ignore(run ~fatal:true prog (args @ [prf^"kernel/"^file;"checker/"^file])))
-  [ "esubst.ml"; "esubst.mli"; "names.ml"; "names.mli" ]
-
 (** * Build the config/Makefile file *)
 
 let write_makefile f =
+
```

Comment:
Why add an empty line here?

Reply:
That's a leftover, once #6959 is merged the whole configure patch will go away. Note however that we will want to split `Coq_config` in two [basic/extended] and `Dune` needs only 2 or 3 configuration variables.

Reply:
Yeah I should rebase this but it is very tricky, specially with the stupid solution we chose to duplicate `.mli` files...

Reply:
Yeah the rebase will have to wait until some of the PRs in the dependency queue get into 8.9